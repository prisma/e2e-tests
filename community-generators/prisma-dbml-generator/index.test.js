const fs = require('fs')

describe('test dbml generator', () => {
  it('should produce same dbml for same Prisma schema', () => {
    const dbml = fs.readFileSync('./dbml/schema.dbml', {
      encoding: 'utf8',
    })
    expect(dbml).toMatchSnapshot()
  })
})
