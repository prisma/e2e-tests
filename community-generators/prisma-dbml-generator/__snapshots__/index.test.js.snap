// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test dbml generator should produce same dbml for same Prisma schema 1`] = `
"//// ------------------------------------------------------
//// THIS FILE WAS AUTOMATICALLY GENERATED (DO NOT MODIFY)
//// ------------------------------------------------------

Table User {
  id String [pk]
  email String [unique, not null]
  name String
}"
`;
